package.json

```diff
@@ -31,8 +32,10 @@
     "winston": "^2.2.0"
   },
   "devDependencies": {
+    "ajv": "^6.0.1",
     "ava": "0.25.0",
     "chai": "4.1.2",
+    "eslint": "^4.9.0",
```

Comment:
Those are dev dependencies, thus, not so critical, but I would definitely hardcode the versions.